R/server.R

```diff
@@ -43,7 +43,7 @@ dolt_server <- function(dir = Sys.getenv("DOLT_DIR", "doltdb"),
                         autocommit = TRUE,
                         read_only = FALSE,
                         log_level = "info",
-                        log_out = NULL,
+                        log_out = "proc.log",
```

Comment:
Should this be the default value or was it for testing?

Reply:
> Should this be the default value or was it for testing?

Sorry that was just for testing. Although it was pretty handy to have it write a default log!

Reply:
Okay removed